.openshift-ci/drivers/pre-build.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM pipeline:src
+FROM quay.io/stackrox-io/apollo-ci:collector-0.3.44-1-gb00ffc52af
```

Comment:
Should this use `pipeline:apollo-ci`?

Reply:
See https://github.com/stackrox/collector/pull/826#discussion_r983375354